### PR TITLE
Cursor based pagination

### PR DESCRIPTION
**Pull request by Yinon-Hever**

### CODE REVIEW

File: src/fireblocks-sdk.ts [hunk 1]
```diff
@@ -272,7 +273,29 @@ export class FireblocksSDK {
...
+    }
+
+    /**
+     * Gets a list of transactions per page matching the given filter, `orderBy` field not allowed
+     * @param filter.before Only gets transactions created before a given timestamp (in milliseconds)
+     * @param filter.after Only gets transactions created after a given timestamp (in milliseconds)
+     * @param filter.status Only gets transactions with the spcified status
+     * @param filter.limit Limit the amount of returned results. If not specified, a limit of 200 results will be used
+     */
+    public async getTransactionsWithPageInfo(filter: TransactionFilter): Promise<TransactionPageResponse> {
```
Comment (yarinvak) [inline src/fireblocks-sdk.ts:286]: create an interface without the `orderBy`

File: src/api-client.ts [hunk 2]
```diff
@@ -5,17 +5,29 @@ import { RequestOptions } from "./types";
 export class ApiClient {
     constructor(private authProvider: IAuthProvider, private apiBaseUrl: string) { }
 
-    public async issueGetRequest(path: string) {
+    public async issueGetRequest(path: string, pageMode: boolean = false) {
```
Comment (yarinvak) [inline src/api-client.ts:8]: consider creating a separate function for this specific logic

Comment (Yinon-Hever) [reply src/api-client.ts:8]: The tradeoff here is to duplicate same code as a result of `resolveWithFullResponse: true` on line 17 and write Get func specific for page or edit all func's using `issueGetRequest` and extract body,  BTW `requestPromise` lib deprecated, so we can fix it when we use any replacement lib.
@tomervil, what approach do u think we need to take here?

Comment (Yinon-Hever) [reply src/api-client.ts:8]: But i totally agree that isn't ideal at all, just a quick win

**Comment by Yinon-Hever** [comment]
@yarinvak and i spoke offline and agreed not to break the SDK, as a result of different behavior on the API vs SDK

**Pull request merged**